### PR TITLE
Updating the Allowed caches for LIGO

### PR DESCRIPTION
**Pull request by efajardo**

### CODE REVIEW

File: virtual-organizations/LIGO.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ AppDescription: Large data intensive analysis and simulation applications involv
   workflows and BOINC technology
 CertificateOnly: false
 Community: The LIGO Scientific Collaboration
-Contacts:
+C1;95;0contacts:
```
Comment (brianhlin) [inline virtual-organizations/LIGO.yaml]: This looks unintended, could you revert this change?

File: virtual-organizations/LIGO.yaml [hunk 2]
```diff
@@ -93,3 +93,18 @@ DataFederations:
...
+      - Stashcache-Manhattan
+      - Stashcache-Sunnyvale
+      - Stashcache-Chicago
+      - Stashcache-Kansas
+      - T1_STASHCACHE_CACHE
+      - PIC_STASHCACHE_CACHE
+
+
+
+
```
Comment (brianhlin) [inline virtual-organizations/LIGO.yaml]: Since you need to make another change in this file, could you get rid of this extra whitespace?

**Review by brianhlin — COMMENTED** [review]
Just some minor changes needed

**Review by djw8605 — APPROVED** [review]
LGTM

**Review by djw8605 — APPROVED** [review]
Wrong option box, Approve!
